CodeToReview.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+
+
+   O | O | O
+  -----------
+   O | O | O
+  -----------
+   O | O | O
+
+
+ */
+package tictactoe;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+    /**
+     * @param args the command line arguments
+     * @throws java.lang.InterruptedException
+     */
+    public static void main(String[] args) throws InterruptedException {
+        Scanner input = new Scanner(System.in);
+        String a1 = "1";
+        String a2 = "2";
+        String a3 = "3";
+        String a4 = "4";
+        String a5 = "5";
+        String a6 = "6";
+        String a7 = "7";
+        String a8 = "8";
+        String a9 = "9";
+        int inO;
+        int inX = 0;
+        boolean winner = false;
+        PrintBoard(a1,a2,a3,a4,a5,a6,a7,a8,a9);
+        System.out.printf("How to play - wait for your turn and when it comes write the number of the field that you want to draw in (from 1 to 9)\n");
```

Comment:
Avoid using print statements in the code. System print degrades the performance of the process significantly.
Using logging frameworks like log4j instead.